modules/services/monitoring/prometheus-node-exporter.nix

```diff
@@ -0,0 +1,81 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    concatStringsSep
+    escapeShellArgs
+    getExe
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOption
+    types
+  ;
+
+  cfg = config.services.prometheus-node-exporter;
+in {
+  options = {
+    services.prometheus-node-exporter = {
```

Comment:
Let’s make this `services.prometheus.exporters.node` to align with NixOS and increase configuration portability. We’ll need `(lib.mkRemovedOptionModule [ "services" "prometheus" "exporters" "node" "openFirewall" ] "No nix-darwin equivalent to this NixOS option.")`, and the same for `firewallRules` and `firewallFilter`. No need to share implementation for now, I think, since NixOS has a bunch of layers of abstraction over their Prometheus exporters that we might not want to pull in immediately and it won’t be a breaking change if we do later.

Reply:
Done